### PR TITLE
OCP4: IA-5(2): PKI-based auth in the cluster

### PR DESCRIPTION
**Pull request by jhrozek**

### CODE REVIEW

**Comment by jhrozek** [comment]
@mrogers950 can you take a look, please?

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 1]
```diff
@@ -556,28 +556,45 @@
...
-  implementation_status: planned
+  implementation_status: complete
   narrative:
     - key: a
       text: |-
-        A control response is planned. Engineering progress can be tracked via:
-
-        https://issues.redhat.com/browse/CMP-327
+        The certificates used for in-cluster mTLS authentication
+        are issued by an internal CA, at the moment it is not possible
```
Comment (mrogers950) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml:564]: nit: we might not want to say 'at the moment' here, it's hard to imagine it being supported anytime soon so not to give that impression :P

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml [hunk 2]
```diff
@@ -556,28 +556,45 @@
...
+        services:
+        https://docs.openshift.com/container-platform/4.6/security/certificate_types_descriptions/service-ca-certificates.html
+
+        When setting up IDPs, all compliant IDP allow the CA that signed
+        the IDP certificate to be set:
+        https://docs.openshift.com/container-platform/4.7/rest_api/config_apis/oauth-config-openshift-io-v1.html
     - key: b
       text: |-
-        A control response is planned. Engineering progress can be tracked via:
+        Most of the private keys are stored in Kubernetes secrets
```
Comment (mrogers950) [inline openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml:582]: nit: Most would imply there are still some that are not protected, so we might need to find and mention the exceptions.

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml (hunk 1 above)
Comment (jhrozek) [reply openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml]: ha, removed

File: openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml (hunk 2 above)
Comment (jhrozek) [reply openshift-container-platform-4/policies/IA-Identification_and_Authentication/component.yaml:593]: I didn't list the exceptions one-by-one, but added a more generic blob based on CIS 1.1.21, what do you think?

**Pull request merged**